starter-core/src/main/java/io/micronaut/starter/feature/oraclecloud/OracleCloudAutonomousDatabase.java

```diff
@@ -78,33 +82,63 @@ public String getCategory() {
         return Category.CLOUD;
     }
 
+    @Override
+    public boolean embedded() {
+        return false;
+    }
+
+    @Override
+    public String getJdbcUrl() {
+        return null;
+    }
+
+    @Override
+    public String getR2dbcUrl() {
+        return null;
+    }
+
+    @Override
+    public String getDriverClass() {
+        return null;
+    }
+
+    @Override
+    public String getDefaultUser() {
+        return "";
+    }
+
+    @Override
+    public String getDefaultPassword() {
+        return "";
+    }
+
+    @Override
+    public String getDataDialect() {
+        return "ORACLE";
+    }
+
     @Override
     public void processSelectedFeatures(FeatureContext featureContext) {
+        super.processSelectedFeatures(featureContext);
         if (!featureContext.isPresent(OracleCloudSdk.class)) {
             featureContext.addFeature(oracleCloudSdkFeature);
         }
     }
 
+    @Override
+    public Map<String, Object> getAdditionalConfig() {
+        Map<String, Object> config = new LinkedHashMap<>(2);
+        config.put("datasources.default.ocid", "");
+        config.put("datasources.default.walletPassword", "");
+        return config;
+    }
+
     @Override
     public void apply(GeneratorContext generatorContext) {
         generatorContext.addDependency(Dependency.builder()
                 .compile()
-                .groupId("io.micronaut.oraclecloud").artifactId("micronaut-oraclecloud-atp")
+                .groupId("io.micronaut.oraclecloud")
```

Comment:
the groupId could be externalised this into static property and used by OracleCloudSdk